spec/support/object_formatter.rb

```diff
@@ -0,0 +1 @@
+RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length = 1000
```

Comment:
This clashes with https://github.com/dxw/DataSubmissionServiceAPI/blob/develop/spec/support/export_spec_expectation_truncations.rb - we should either do it that way, or this way, but not both!